components/layout_2020/flow/mod.rs

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 #![allow(rustdoc::private_intra_doc_links)]
+#![allow(clippy::too_many_arguments)]
```

Comment:
can we allow `too_many_arguments ` on project level?

Reply:
It would be nice to avoid adding functions with too many arguments in the future though.

Reply:
I agree with @mrobinson, I was thinking that once the clippy warning were under control we could add a clippy check to `test-tidy` or the ci, so that new code was already linted, in a similar way it does with formatting. It's hard to change previous functions but it would be nice if new ones were more manageable.

Reply:
Should then this be allowed for the specific functions instead of to the entire file?

Reply:
Yes, I think that makes the most sense, even if it is more verbose for now. I'll change it!